packages/web/src/components/ui/BarChart/BarChart.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useMemo } from 'react';
+import styles from './BarChart.module.scss';
+import clsx from 'clsx';
+import { formatNumber } from '../../../utils/helpers';
+import { Icon } from '../Icon/Icon';
+
+type Bar = {
+  fill: string;
+  uses: number;
```

Comment:
nit: This component is a good candidate to become a more generic one by splitting off the text description into children (the `moduleVersion`, `vulnerabilities` and `uses` props). And then making a more-specialized `ModuleUsageBar` component.